plugin-server/tests/main/process-event.test.ts

```diff
@@ -1475,6 +1476,184 @@ test('distinct with anonymous_id which was already created', async () => {
     expect(person.is_identified).toEqual(true)
 })
 
+test('distinct with anonymous_id which was already created triggers foreign key updates', async () => {
```

Comment:
hmm, should add a test in django-land for where old user is identified, which means we won't reach this code path, and no updates would occur. Will the override django code handle that?